purchase_incoterm/models/res_partner.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    purchase_incoterm_id = fields.Many2one(
+        comodel_name="account.incoterms",
+        ondelete="restrict",
+        string="Purchase Incoterm",
```

Comment:
Use the same terms than `sale_partner_incoterm`:
```suggestion
        string="Default Purchase Incoterm",
        help="The default incoterm for new purchase orders for this supplier.",
```

Reply:
Thanks for reivew, branch updated